### PR TITLE
OCM-4458 | test: automate id:73755 validation for create cluster with external_auth_config

### PR DESCRIPTION
**Pull request by aaraj7**
output log :- https://privatebin.corp.redhat.com/?9cdb7dc44e59429b#8ScK6rdMGptkhN8AR1crGbeZssKvpRertJdKxuxpvee2

### CODE REVIEW

**Comment by yasun1** [comment]
@aaraj7 Due to the label issue, the case is not picked up in the presubmit testing. We will do an enhancement [here](https://github.com/openshift/release/pull/52345/files). To not block this merge, please paste the local log in https://privatebin.corp.redhat.com, and attach its link here.

File: tests/e2e/hcp_external_auth_test.go [hunk 1]
```diff
@@ -243,5 +246,67 @@ var _ = Describe("Rosacli Testing", func() {
...
+						"--operator-role-prefix": operatorPrefix,
+					}
+					rosalCommand.ReplaceFlagValue(replacingFlags)
+					rosalCommand.AddFlags("--dry-run", "--external-auth-providers-enabled", "-y")
+					output, err := rosaClient.Runner.RunCMD(strings.Split(rosalCommand.GetFullCommand(), " "))
+					Expect(err).To(HaveOccurred())
+					Expect(output.String()).To(ContainSubstring("ERR: External authentication configuration is only supported for a Hosted Control Plane cluster."))
+				}
+
+				if isHostedCP {
```
Comment (radtriste) [inline tests/e2e/hcp_external_auth_test.go:278]: `else` instead ?

**Review by radtriste — COMMENTED** [review]
small comment on code, otherwise looks good

File: tests/e2e/hcp_external_auth_test.go (hunk 1 above)
Comment (aaraj7) [reply tests/e2e/hcp_external_auth_test.go]: I used else instead, Thanks

**Pull request merged**